### PR TITLE
doc: warnings to notes

### PR DESCRIPTION
**Pull request by peknis**
Changed warnings to notes. Includes also general language cleanups.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst [hunk 1]
```diff
@@ -37,14 +37,14 @@ In such case, this minimal implementation includes only a call to the default si
 
 With this call, the device will be able to join the Zigbee network or join a new network when it leaves the previous one.
 
-In general, using the default signal handler is worth considering because of the following reasons:
+In general, using the default signal handler is worth considering because:
```
Comment (ru-fu) [inline doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst:40]: We should have a full sentence here, so I prefer to leave as is.

**Review by ru-fu — CHANGES_REQUESTED** [review]
I agree with (almost ;) ) all of the changes, but there are quite a few changes that are unrelated to changing warnings to notes.
It would be difficult to sort them into two different commits, but please edit the commit text to make it clear that it also includes general language cleanups.

File: doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst (hunk 1 above)
Comment (peknis) [reply doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst]: I don't fully agree, but I changed this back as it was.

**Comment by ru-fu** [comment]
Can you edit the commit text as well (see above)?

**Comment by peknis** [comment]
> Can you edit the commit text as well (see above)?

I tried, but no success there from the git bash, so I edited it manually in the PR.

**Comment by ru-fu** [comment]
> 
> 
> > Can you edit the commit text as well (see above)?
> 
> I tried, but no success there from the git bash, so I edited it manually in the PR.

That won't help. ;) You can only manually edit the text for the PR, but the updated text needs to go in the commit description.
If you do a commit --amend, you can edit the text in the editor.

**Comment by peknis** [comment]
> > > Can you edit the commit text as well (see above)?
> > 
> > 
> > I tried, but no success there from the git bash, so I edited it manually in the PR.
> 
> That won't help. ;) You can only manually edit the text for the PR, but the updated text needs to go in the commit description.
> If you do a commit --amend, you can edit the text in the editor.

That is exactly what I did, but for some reason, it did not work. I'll try again.

**Comment by peknis** [comment]
Better now?

**Review by ru-fu — APPROVED** [review]

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -949,7 +949,7 @@ To use the nRF Desktop application with your custom board:
      * Each of the logical LEDs can have either one (monochromatic) or three color channels (RGB).
        Such color channel is a physical LED.
      * The project uses Pulse-Width Modulation (PWM) channels to control the brightness of each physical LED.
-       The PWM peripheral must be configured in DTS files, and the :file:`_def` file of the LEDs module must be configured to indicate which PWM channel is assigned to each LED color.
+       Configure the PWM peripheral in DTS files, and the :file:`_def` file of the LEDs module must be configured to indicate which PWM channel is assigned to each LED color.
```
Comment (b-gent) [inline applications/nrf_desktop/README.rst:952]: This is now in active mode in first part and passive mode in 2nd  :) 

```suggestion
       Configure the PWM peripheral in DTS files, and configure the :file:`_def` file of the LEDs module to indicate which PWM channel is assigned to each LED color.
```

**Review by b-gent — CHANGES_REQUESTED** [review]

File: applications/nrf_desktop/README.rst (hunk 2 above)
Comment (peknis) [reply applications/nrf_desktop/README.rst:952]: Good catch. I noticed this myself, but left it for you ;)

**Review by b-gent — APPROVED** [review]

**Pull request merged**